src/PriorityQueue/index.js

```diff
@@ -0,0 +1,69 @@
+const validateNumber = require('../validation/number');
+
+class PriorityQueue {
+	constructor() {
+		this.queueArr = [];
+	}
+
+	// show the current queue
+	queue() {
+		return this.queueArr.join(' ');
+	}
+
+	// enqueue method
+	push(elem) {
+		validateNumber(elem, 'PriorityQueue.push');
+
+		let flag = false;
+
+		for (let i in this.queueArr) {
+			if (this.queueArr[i] > elem) {
+				this.queueArr.splice(i, 0, elem);
+				flag = true;
+				break;
+			}
+		}
+
+		if (!flag) {
+			this.queueArr.push(elem);
+		}
+
+		return true;
+	}
+
+	// dequeue
+	pop() {
+		if (this.queueArr.length === 0) {
+			return 'Queue Underflow';
+		}
+		return this.queueArr.shift();
```

Comment:
@thisisabdus `priority queue` should by default return the maximum value, and since you're arranging it in increasing order, the *pop* method should return the last value (_which is the maximum_); so it should be - `return this.queueArr.pop();`